src/url-params.ts

```diff
@@ -27,6 +28,8 @@ export class AnalyticsUrlParams {
 		currentDeviceId: string | null,
 	) {
 		const list = inputIdString ? inputIdString.split(deviceIdSeparator) : [];
+		this.passedDeviceIds = new Set(Array.from(list));
```

Comment:
`list` is already an array (as per 1 line above) so not sure if the extra conversion is of any need?